app/src/main/java/ch/sdp/vibester/activity/BuzzerScreenActivity.kt

```diff
@@ -114,10 +112,10 @@ class BuzzerScreenActivity : GameActivity() {
     private fun startRound(ctx: Context, gameManager: BuzzerGameManager) {
         gameIsOn = true
         findViewById<LinearLayout>(R.id.answer).visibility=View.INVISIBLE
-
-        // fetch song and initialise answerText. We'll see later for the image
         val title = gameManager.getCurrentSong().getTrackName()
-        findViewById<TextView>(R.id.answerText).text=title
+        val artist = gameManager.getCurrentSong().getArtistName()
+        findViewById<TextView>(R.id.songTitle).text= "$title - $artist"
+        Glide.with(ctx).load(gameManager.getCurrentSong().getArtworkUrl()).override(artworkDim, artworkDim).into(findViewById(R.id.songArtwork))

```

Comment:
Oh I have never heard of that Library. After having a look on internet, it may be great to use it everywhere. I will check later in my refactor if it is worth it or not.